tests/api/crud/model_monitoring/test_deployment.py

```diff
@@ -105,3 +106,41 @@ def test_credentials(
                 stream_path="kafka://stream",
                 tsdb_connection="taosws://",
             )
+
+    @pytest.mark.skipif(
+        os.getenv("V3IO_FRAMESD") is None or os.getenv("V3IO_ACCESS_KEY") is None,
+        reason="Configure Framsed to access V3IO store targets",
+    )
+    def test_cred_v3io(
+        self, monitoring_deployment: mm_dep.MonitoringDeployment, store_connection: str
+    ) -> None:
+        with pytest.raises(mlrun.errors.MLRunBadRequestError):
+            monitoring_deployment.check_if_credentials_are_set(
+                only_project_secrets=True
+            )
+
+        with pytest.raises(mlrun.errors.MLRunInvalidMMStoreType):
+            monitoring_deployment.set_credentials(
+                endpoint_store_connection="v3io",
+                stream_path="kafka://stream",
+                tsdb_connection="promitheus",
```

Comment:
Is this typo on purpose to get an error? Better be explicit about it. Something like:
```suggestion
                tsdb_connection="wrong",
```